### PR TITLE
Use issue icon in RHS

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary

| Before  | After           |
|---------|-----------------|
| Screenshot from 2020-09-25 20-29-24 | Screenshot from 2020-09-25 20-26-52 |
| Screenshot from 2020-09-25 20-29-49| Screenshot from 2020-09-25 20-27-08 |

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-github/issues/375

### CODE REVIEW

**Review by asaadmahmood — APPROVED** [review]

**Review by jfrerich — APPROVED** [review]

File: webapp/src/components/sidebar_right/github_items.jsx [hunk 1]
```diff
@@ -34,9 +35,24 @@ function GithubItems(props) {
...
 
         if (item.number) {
+            let iconType;
+
+            // item is a pull request
+            if (item.pull_request) {
+                iconType = GitPullRequest;
+            } else {
+                iconType = IssueOpened;
+            }
```
Comment (larkox) [inline webapp/src/components/sidebar_right/github_items.jsx:45]: Nit:
```suggestion
            let iconType = IssueOpened;

            // item is a pull request
            if (item.pull_request) {
                iconType = GitPullRequest;
            }
```

File: webapp/src/components/sidebar_right/github_items.jsx [hunk 2]
```diff
@@ -34,9 +35,24 @@ function GithubItems(props) {
...
             number = (
                 <strong>
-                    <i className='fa fa-code-fork'/>{' #' + item.number}
+                    <div style={{...style.icon}}>
```
Comment (larkox) [inline webapp/src/components/sidebar_right/github_items.jsx:48]: Nit: wouldn't it make more sense here a `<span>` instead of a `<div>`?

**Review by larkox — APPROVED** [review]
LGTM, just two nits.

File: webapp/src/components/sidebar_right/github_items.jsx (hunk 2 above)
Comment (hanzei) [reply webapp/src/components/sidebar_right/github_items.jsx]: Good suggestion :+1:

File: webapp/src/components/sidebar_right/github_items.jsx (hunk 1 above)
Comment (hanzei) [reply webapp/src/components/sidebar_right/github_items.jsx:45]: What I like about the current style is that it makes the code more readable IMO. It's clear that  it's an either/or and the reader doesn't have to check if `iconType` is defined.

But that is very much a subjective opinion.

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Confirmed the icons have been replaced as expected
- Link to Issue and PR are functional to navigate to their respective issues
- Tested with multiple themes to ensure no updates needed here
LGTM!

Thanks @hanzei  this is a nice little piece of polish!

**Pull request merged**